lpa-submissions-e2e-tests/cypress/support/common/completeQuestionnaire.js

```diff
@@ -0,0 +1,17 @@
+module.exports = () => {
+  // Visit task list page to invoke session, which generates an appeal reply ID
+  cy.goToTaskListPage();
+
+  let id
+  cy.getAppealReplyId().then(replyId => id = replyId);
+
+  cy.fixture('completedAppealReply.json').then((reply) => {
+    cy.request(
+      'PUT',
+      `${Cypress.env('appealReplyServiceBaseUrl')}/api/v1/reply/${id}`,
+      reply
+    ).then(response => {
+      expect(response.body.aboutAppealSection.submissionAccuracy).to.have.property('accurateSubmission', false);
+    });
+  });
+};
```

Comment:
observation not complaint:
you get the speed out of this approach; you do get some inherent risk as well.

without a test that does something like:
* manually fill in the form via the browser and submit it
* post the form data to the api and submit it
* confirm that the same stuff comes out when we submit (message to horizon/pdfs/emails/whatever)
...you run the risk that someone changes the browser flow in some way and your test-data that you're posting in gets left behind... this can give you a place where your tests continue to run green while the actual expected behaviour isn't exhibited due to stale test data

Reply:
Agreed, as discussed before, we will be testing for this outside the realm of this ticket